Pipeline/PackageBuildOutputs/utilities/sbomGenerator.groovy

```diff
@@ -0,0 +1,206 @@
+@groovy.transform.BaseScript com.ibm.dbb.groovy.ScriptLoader baseScript
+import java.io.File
+import com.ibm.dbb.build.CopyToHFS
+import com.ibm.dbb.build.BuildProperties
+import com.ibm.dbb.build.DBBConstants
+import com.ibm.dbb.build.DBBConstants.CopyMode
+import com.ibm.dbb.build.report.BuildReport
+import com.ibm.dbb.build.report.records.*
+import groovy.transform.*
+import groovy.cli.commons.*
+import java.nio.file.*
+import static java.nio.file.StandardCopyOption.*
+import com.ibm.jzos.ZFile
+import org.cyclonedx.*
+import org.cyclonedx.model.*
+
+/************************************************************************************
+ * This utility script helps generating an SBM from the DBB BuildReport
+ *
+ * Version 1 - 04/2024
+ *  Initial implementation of SBOM Generation
+ ************************************************************************************/
+
+
+// Property prefix, matching what is defined in zAppBuild/build.groovy
+@Field String hashPrefix = ':githash:'
+@Field bomRef = 0
+
+// SBOM generation-related objects
+@Field HashSet<String> applications
+@Field HashMap<String, HashMap<String, String>> outputsInSBOM
+@Field ArrayList<Dependency> sbomDependencies
+@Field Bom sbom
+
+def initializeSBOM(String sbomAuthor) {
+	// HashMap of files to report in SBOM
+	applications = new HashSet<String>()
+	outputsInSBOM = new HashMap<String, HashMap<String, String>>()
```

Comment:
I only see two occurrences of `outputsInSBOM`, but nothing it added to the map. Is this code unused?

Reply:
same with applications btw.

Reply:
Done